src/components/ItemFilters/README.md

```diff
@@ -1,21 +0,0 @@
-# Item Filters
```

Comment:
Keep this around but update it based on the new implementation. I think it's useful but if it's straightforward and not useful then do remove it.

Reply:
That sounds good to me. Mind if i open a ticket and work the docs as a follow-up when the full implementation is complete? I can leave the file in for now with a Todo comment.

Reply:
It shouldn't be a separate ticket but part of the work for the implementation ticket.